year-pages/2019-conferences-list.md

```diff
@@ -10,4 +10,4 @@
 | [RubyConf India](https://www.rubyconfindia.org/) | 20-21 Jan | Resort Rio and Club Rio Royale, Goa | A global event complementing other RubyConf events across the world. This year it's a single track 2 day event, focused on the Language, Framework and Tools | [Yes](https://www.rubyconfindia.org/scholarship/) |
 | [Pragma IIITA](http://pragmaconf.tech/) | 19 -20 Jan | Indian Insitute of Information Technology, Allahabad | Pragma is premier student design and developers’ conference held at IIIT-Allahabad. Attracting some of the best student developers from across India, Pragma is a 3 day extravaganza featuring a hackathon, workshops and numerous talks. | No
 | [GopherCon](https://gopherconindia.com/) | 18-19 Jan | Resort Rio, Goa | Go conference in India. Go is an open source project developed by a team at Google and many contributors for the open source community | [Yes]( https://gopherconindia.com/scholarship/) |
-| [JSFoo](https://jsfoo.in/2019-pune/) | 10 Jan | MCCIA Trade Tower, Pune | The second Pune edition of India’s premier JavaScript conference | No |
+| [JSFoo Pune](https://jsfoo.in/2019-pune/) | 10 Jan | MCCIA Trade Tower, Pune | The second Pune edition of India’s premier JavaScript conference | No |
```

Comment:
Organizers seemed to have named this as `JSFoo Pune` instead of `JSFoo` in the website. I'm not sure if there is a difference between them, but I agree that `JSFoo` sounds better than `JSFoo Pune`.

Let's update this again later if there is another version of the conference which is just `JSFoo`.